spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -297,6 +298,12 @@
 				<version>${r2dbc-postgres-driver.version}</version>
 			</dependency>
 
+			<dependency>
+				<groupId>com.google.cloud</groupId>
+				<artifactId>cloud-spanner-r2dbc</artifactId>
+				<version>${cloud-spanner-r2dbc.version}</version>
```

Comment:
I think it's better to move this up and next to our starters, perhaps at [L253](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/917e0f3293d58826c2b0053023b27f9d285b6444/spring-cloud-gcp-dependencies/pom.xml#L253).
And the version here should be `${project.version}` so the released bom does not have lagged versions.

Reply:
Reminder from in person: Add the dialect module here. and do not depend on `spring-cloud-gcp-dependencies` in the actual module.

Reply:
updated both as discussed